client/src/app/pages/volunteer/volunteer.component.html

```diff
@@ -17,7 +17,7 @@ <h3>Volunteering at Events:</h3> <p> We always need help running competitions sm
 
       <h3>Community Outreach:</h3> <p> If you enjoy sharing your passion for cubing with others, perhaps you could help us with demonstrations at schools or community centers.  While we aren’t actively seeking out these opportunities, we would fully support and provide resources for any initiatives you start. </p>
 
-      <h3>Website Design:</h3> <p> If you have skills or interest in software development or graphic design, we have several tasks that might interest you. You can find these listed on our GitHub page: CubingZA Website Issues. A specific area where we could use help is improving the design of our competition announcement emails. </p>
+      <h3>Website Design:</h3> <p> If you have skills or interest in software development or graphic design, we have several tasks that might interest you. You can find these listed on our GitHub page: CubingZA Website Issues. A specific area where we could use help is improving the design of our competition announcement emails. A great place to start is looking through the github repository and finding an issue to work on that interests you! Click <a href="https://github.com/CubingZA/CubingZA-Website/issues">Here!</a></p>
```

Comment:
```suggestion
      <h3>Website Design:</h3> <p> If you have skills or interest in software development or graphic design, we have several tasks that might interest you. You can find these listed on our GitHub page: <a href="https://github.com/CubingZA/CubingZA-Website/issues">CubingZA Website Issues</a>.</p>
```

The bit about announcement emails isn't too relevant on the website because you're already working on that. The bit at the end is a bit redundant. You already mention the issues page, so it's better to link that directly.